presto-plugin-toolkit/src/test/java/com/facebook/presto/plugin/base/security/TestFileBasedAccessControl.java

```diff
@@ -114,12 +118,21 @@ private static ConnectorIdentity user(String name)
     private ConnectorAccessControl createAccessControl(String fileName)
             throws IOException
     {
-        String path = this.getClass().getClassLoader().getResource(fileName).getPath();
         FileBasedAccessControlConfig config = new FileBasedAccessControlConfig();
-        config.setConfigFile(path);
+        config.setConfigFile(getResourceFile(fileName).getPath());
         return new FileBasedAccessControl(config);
     }
 
+    private static File getResourceFile(String resourceName)
```

Comment:
can both getResourceFile method be shared?

Reply:
> can both getResourceFile method be shared?

Hi @jainxrohit , I did think about that but the same method used in two different components. So you can either let them reference the other or move the code to a 3rd component (we do have a similar function in presto-main) and point the two component to it. But that would involve adding one extra dependency on something that only has a very small thing we want to use. Also in the presto-main case, that would cause a cyclic dependency in my testing. 
So I decided later to duplicate the four lines of code in each component. Do you have something in mind where we should put this kind of utility methods that can be shared among comments w/o complicating things like dependencies ?